server/sonar-web/src/main/js/apps/projects/filters/QualityGateFilter.js

```diff
@@ -21,28 +21,30 @@ import React from 'react';
 import FilterContainer from './FilterContainer';
 import Level from '../../../components/ui/Level';
 
-export default class QualityGateFilter extends React.Component {
+export default class QualityGateFilter extends React.PureComponent {
   renderOption = (option, selected) => {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove curly braces and "return" from this arrow function body. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)